DeAceroDLabs/marketplace-frontend
# [DMP-665] Add "add to cart" button

## What is done

[[DMP-665] Agregar botón de agregar al carrito](https://deacero.atlassian.net/browse/DMP-665)

## How is done

1. Modified some styles on the card component.
2. Added a new card on the ProductDetailView component.
3. Added a Button inside the new card.

## How to test

1. Run the project via `npm start`
4. Click on any product from the list on the home page.
5. You should see the new card and the button at the top right of the page.
6. Clicking the button should do nothing.

## Evidence

Add images or screen  recordings on how the feature was working after and before the fix (if applies)
